distribution/src/main/resources/config.yml

```diff
@@ -11,7 +11,7 @@ hades2:
     debug: false
 
 hades2-default-parameters:
-    dcMode: true
+    dcMode: false
```

Comment:
Why did you change default configuration of iTools configuration ?

Reply:
reverted